.circleci/config.yml

```diff
@@ -97,11 +97,11 @@ commands:
       - when:
           condition: <<parameters.tf1>>
           steps:
-            - run: docker pull determinedai/environments:py-3.7-pytorch-1.7-tf-1.15-cpu-606fd02
+            - run: docker pull determinedai/environments:py-3.7-pytorch-1.7-tf-1.15-cpu-da845fc
```

Comment:
Looks like this actually includes @shiyuann 's latest changes, as of "add a github template". Looks like there was a change that got reverted to. Let's have him confirm this is intended for release already.

Reply:
I think `c1b75818ca76628196c5939f80a6ed4912247db1` and `da845fc89f8ff4b03e2018e407ebf5ff545b3067` should both work.

Reply:
`254d51127cc540b30a2016081e3e5b4bffa8d292` work too

Reply:
his change wasn't intended for that release, but it overwrote 0.15.0, so we've reverted it to overwrite 0.15.0 back, and added a harmless template. so all is correct in my opinion: last material change what the one with horovod cpu.

Reply:
I want to use the latest so the "hashed" and "versioned" are the same image.